app/views/missing/report.php

```diff
@@ -195,7 +195,7 @@
                                     </div>
                                     <div class="panel-body" id="panelBody4">
                                         <label for="w_phone_no">Phone number </label>
-                                        <input id="w_phone_no" class="form-control" placeholder="Enter Phone Number" name="contact_whom[phone_no]" required /><br>
+                                        <input id="w_phone_no" class="form-control" placeholder="Enter Phone Number" name="contact_whom[phone_no]" pattern="\d{10}" required /><br>
```

Comment:
It should be checked on both the sides for better user experience. It should show message before submitting, that is only possible if we do it here. Also, if someone is trying to fiddle with the system, we should not allow that, that is to be done on back-end with PHP.